prison/src/src/Data/Schedule.java

```diff
@@ -0,0 +1,80 @@
+package src.Data;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+
+import java.util.ArrayList;
+
+public class Schedule {
+    public ArrayList<Activity> activities = new ArrayList<>();
+    public ObservableList<String> activityNames = FXCollections.observableArrayList();
+    public ObservableList<Guard> guards = FXCollections.observableArrayList();
+    public ObservableList<Area> areas = FXCollections.observableArrayList();
+    public ObservableList<Groep> prisonGroeps = FXCollections.observableArrayList();
+
+    public Schedule(){
+        {
```

Comment:
Waarom zit er een code block in een code block?
```suggestion
    public Schedule() {
```

Reply:
+1